.github/workflows/build.yml

```diff
@@ -175,7 +175,7 @@ jobs:
         uses: nttld/setup-ndk@v1
         id: setup-ndk
         with:
-          ndk-version: r28
+          ndk-version: r28b
```

Comment:
IIRC, this should also be reflected here: https://github.com/servo/mozjs/blob/1410a4bafe4674ea17b1c4f0053a3e589ebce989/android-build#L15

Reply:
That file ignores any suffix.